src/openapi/monitor.yml

```
@@ -0,0 +1,155 @@
+model:
+  SimpleListItem:
+    type: object
+    properties:
+      name:
+        type: string
+        maxLength: 250
+
+  BareMetalPowerMetric:
+    type: object
+    properties:
+      platform:
+        type: string
+        maxLength: 50
+      all:
+        type: integer
+      on:
+        type: integer
+      off:
+        type: integer
+      wattage_on:
+        type: integer
+      wattage_off:
+        type: integer
+
+  UsageMetric:
+    type: object
+    properties:
+      platform:
+        type: string
+        maxLength: 20
+      all:
+        type: integer
+      requested:
+        type: integer
+      provisioned:
+        type: integer
+      available:
+        type: integer
+      cpus_available:
+        type: integer
+      memory_available:
+        type: integer
+      cpus_used:
+        type: integer
+      memory_used:
+        type: integer
+
+
+endpoints:
+  bm_hosts_to_monitor:
```

I think you need to add the operation ([RHUB - Code Etiquette Agreement](https://docs.google.com/document/d/1JMWj7Vh6bJeo16kXzCzZvHvHrF34ZgLpF0vwnKSf7cE/edit?usp=sharing)).

Like:
```suggestion
  bm_hosts_to_monitor_list:
```

---

Right. Changed others to _get then, it's indeed more clear.